src/constants/mod.rs

```diff
@@ -0,0 +1,117 @@
+
+// Re-export all constants from their respective modules
+pub use fs_constants::*;
+pub use net_constants::*;
+pub use sys_constants::*;
+pub use vmmap_constants::*;
+
+// ===== Memory Protection Flags =====
+// Source: include/uapi/asm-generic/mman-common.h
+// These flags control memory segment permissions for mmap and mprotect
+pub const PROT_NONE: i32 = 0x0;    // No permissions - pages may not be accessed
+pub const PROT_READ: i32 = 0x1;    // Pages may be read
+pub const PROT_WRITE: i32 = 0x2;   // Pages may be written
+pub const PROT_EXEC: i32 = 0x4;    // Pages may be executed
+pub const PROT_MASK: u32 = 0x7;    // Mask for all permission bits
+                                   // Note: Some platforms may support additional bits
+
+// ===== Memory Mapping Flags =====
+// Source: include/uapi/asm-generic/mman.h, include/uapi/asm-generic/mman-common.h
+// These flags control the behavior of mmap operations
+pub const MAP_SHARED: i32 = 0x01;         // Share changes with other processes
+pub const MAP_PRIVATE: i32 = 0x02;        // Changes are private to this process
+pub const MAP_FIXED: i32 = 0x10;          // Interpret address exactly
+pub const MAP_ANON: i32 = 0x20;           // Don't use a file descriptor
+pub const MAP_ANONYMOUS: i32 = MAP_ANON;   // Synonym for MAP_ANON
+pub const MAP_SHARING_MASK: i32 = 0x03;    // Mask to isolate sharing bits
+                                          // Note: Values may differ on BSD systems
+
+// ===== Memory Mapping Error Values =====
+// Source: include/uapi/asm-generic/mman-common.h
+// Standard error return value for mmap
+pub const MAP_FAILED: *mut std::ffi::c_void = (-1_isize as *mut std::ffi::c_void);
+
+// ===== Memory Page Size Constants =====
+// Note: These values are architecture-dependent
+// Current values are for x86_64 Linux
+pub const PAGESHIFT: u32 = 12;           // 4KB pages (1 << 12 = 4096)
+pub const PAGESIZE: u32 = 1 << PAGESHIFT;
+pub const MAP_PAGESHIFT: u32 = 16;       // Custom value for Lind
+pub const MAP_PAGESIZE: u32 = 1 << MAP_PAGESHIFT;
+
+// ===== Memory Remapping Flags =====
+// Source: include/uapi/asm-generic/mman-common.h
+pub const MREMAP_MAYMOVE: u32 = 0x01;    // Can relocate mapping
+pub const MREMAP_FIXED: u32 = 0x02;      // New address is specified exactly
+
+// ===== File Access Modes =====
+// Source: include/uapi/asm-generic/fcntl.h
+// These flags control how files are opened and accessed
+pub const O_ACCMODE: i32 = 0o003;    // Mask for file access modes
+pub const O_RDONLY: i32 = 0o0;       // Open read-only
+pub const O_WRONLY: i32 = 0o1;       // Open write-only
+pub const O_RDWR: i32 = 0o2;         // Open read-write
+pub const O_CREAT: i32 = 0o100;      // Create file if it doesn't exist
+pub const O_EXCL: i32 = 0o200;       // Error if O_CREAT and file exists
+pub const O_TRUNC: i32 = 0o1000;     // Truncate file to zero length
+pub const O_APPEND: i32 = 0o2000;    // Append mode - writes always at end
+
+// ===== File Types =====
```

Comment:
I'm a bit confused why we keep those here instead of moving to corresponding files?

Reply:
removed them from mod.rs